decisions4s-core/src/test/scala/decisions4s/exprs/AndTest.scala

```diff
@@ -1,26 +1,22 @@
 package decisions4s.exprs
 
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
 import decisions4s.exprs.TestUtils.checkExpression
-import munit.FunSuite
 
-class AndTest extends FunSuite {
-  class AndTest extends FunSuite {
-    test("basic") {
-      checkExpression(And(True, True), true)
-      checkExpression(True and True, true)
-      checkExpression(True && True, true)
-
-      checkExpression(And(True, False), false)
-      checkExpression(True and False, false)
-      checkExpression(True && False, false)
-
-      checkExpression(And(False, True), false)
-      checkExpression(False and True, false)
-      checkExpression(False && True, false)
-
-      checkExpression(And(False, False), false)
-      checkExpression(False and False, false)
-      checkExpression(False && False, false)
-    }
+class ExampleSpec extends AnyFlatSpec with Matchers {
```

Comment:
I would go for FreeSpec tbh, playing with all the should/when/in is more terrible than it's worth.